### PR TITLE
Resource tier support for CLI 

### PR DESCRIPTION
**Pull request by swathimr**
Updates to existing API:
 - Create new edgeworker
 - Edit an EdgeWorker ID
 - Fetch EdgeWorker IDs

New Endpoints
- Clone an EdgeWorker ID
- List Contract IDs for Resource Tiers
- List Resource Tiers for a specific Contract ID
- Fetch the Resource Tier for a specific EdgeWorker ID

### CODE REVIEW

**Comment by mryakan** [comment]
We need to also update the readme as part of this PR.

File: README.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ Commands:
 | validate \| vv `<bundlePath>` | Validates a code bundle version without uploading the code bundle. |
 | create-auth-token \| auth `[options] <secretKey>` | Generates an authentication token that can be used to get detailed EdgeWorker debug response headers. |
 | generate-secret \| secret `[options]` | Generates a secret key that can be used to generate auth token or in property variable. |
+| clone \| clone `<edgeworker-identifier> <resourceTierId> [options]` | Clones an Edgeworker from the existing Edgeworker Id. |
+| list-contracts \| li-contract `[options]` | List of contract ids that user has access to. |
+| list-restier \| li-restier `[options]` | List Resource Tiers that can be used to create or clone EdgeWorker Id. |
+| show-restier \| show-restier `<edgeworker-identifier>` | Customers can get Resource Tier details for a specific EdgeWorker Id. |
```
Comment (surbhi65) [inline README.md:75]: very minor but **show-res-tier** might be easy to understand. same for **list-res-tier**

File: README.md [hunk 2]
```diff
@@ -328,6 +336,58 @@ Generates a random secret key that can be used to create edgeworkers authenticat
 
 Usage: `akamai edgeworkers generate-secret`
 
+### Clone an Edgeworker Id
+Allows customer to clone an Edgeworker from an existing Edgeworker Id.
+
+Usage: `akamai edgeworkers clone <edgeworker-identifier> <resourceTierId> [options]`
+
```
Comment (surbhi65) [inline README.md:343]: what are [options] for?

File: docs/edgekv_cli.md [hunk 3]
```diff
@@ -313,6 +315,29 @@ Example:
...
+ 
+Retrieve an edgeKV access token.
+ 
+Usage:
+`akamai edgekv create token <tokenName> --save_path=<path> --overwrite`
+ 
+| Option | Existence | Description |
+| - | - | - |
+| --save_path | Optional | Path specifying where to save the edgekv_tokens.js token file. We recommend that you save the token file in the same location as the EdgeWorkers code bundle file (.tgz). The EdgeWorkers code bundle is then automatically updated every time this command updates the edgekv_tokens.js token file.If a path is not provided the token value is displayed. This token must be securely stored and manually added to the edgekv_tokens.js token file and EdgeWorkers code bundle. |
+| -o, --overwrite | Optional | This option is used in conjunction with the --save_path option to overwrite the value of an existing token with the same name in the edgekv_tokens.js file. |
```
Comment (surbhi65) [inline docs/edgekv_cli.md:338]: **file.If** missing space before If

Comment (surbhi65) [inline docs/edgekv_cli.md:338]: whats the difference between -- and - parameters like --save_path and -o?

**Review by surbhi65 — APPROVED** [review]

File: docs/edgekv_cli.md [hunk 4]
```diff
@@ -313,6 +315,29 @@ Example:
...
+| Option | Description |
+| - | - |
+| -h, --help  | Display information on how to use this EdgeKV command. |
+ 
+### Retrieve Access Token
+ 
+Retrieve an edgeKV access token.
+ 
+Usage:
+`akamai edgekv create token <tokenName> --save_path=<path> --overwrite`
```
Comment (mryakan) [inline docs/edgekv_cli.md:333]: should be "download" not "create"

**Review by mryakan — CHANGES_REQUESTED** [review]

**Pull request merged**